include/runtime/stackmgr.h

```diff
@@ -127,6 +129,24 @@ class StackManager {
   /// Unsafe leave top label.
   void leaveLabel() { LabelStack.pop_back(); }
 
+  /// Unsafe getter of next instruction to execute.
+  const AST::Instruction *getNextInstr() {
+    if (LabelStack.size() == 0) {
+      return nullptr;
+    }
+    auto &L = LabelStack.back();
+    return (*(L.Curr++)).get();
```

Comment:
You use `++` without checking the boundary.

Reply:
This is unsafe version, which bases on there's always an End instruction before the end of instruction vector.